### ISSUE TITLE
Blank screen on Linux Mint 20.2 (AppImage)

### ISSUE THREAD

**Issue by waynew** [vgc/vgc#567]
Both  ./vgcillustration-alpha-2021-06-15.6-x86_64.AppImage and VPaint-1.7-x86_64.AppImage behave the same way. When I start it up, it just shows whatever was already on the desktop.

Here's what happens:

```
❯ ./vgcillustration-alpha-2021-06-15.6-x86_64.AppImage
QOpenGLWidget: Failed to create context
QOpenGLWidget: Failed to create context
QOpenGLWidget: Failed to create context
QOpenGLWidget: Failed to create context
qt.qpa.backingstore: composeAndFlush: QOpenGLContext creation failed
qt.qpa.backingstore: composeAndFlush: makeCurrent() failed

```

Here's a screenshot of what it looks like:
![vgc](https://user-images.githubusercontent.com/189750/148633162-fb34952b-e1f6-42b6-a745-9ab9732f0110.png)

That's what the Window looks like. Those are just parts of my windows that were visible beforehand.

**Comment by waynew** [comment]
Okay well, I guess this isn't an issue but it's really, *really* weird how I solved it.

I restarted my machine.

That's it. I know that I had restarted my computer since I had installed nVidia/GL drivers, but for some reason it exhibited this behavior.

It's possible that maybe I had done an `apt-get update` but :shrug:.

If anyone else runs into this issue, try restarting and see if that fixes it. I would be *extremely* impressed if that's what solved this issue for others.
